api/internal/handler/consumer/consumer.go

```diff
@@ -105,6 +105,13 @@ func (h *Handler) Create(c droplet.Context) (interface{}, error) {
 	}
 	input.ID = input.Username
 
+	if _, ok := input.Plugins["jwt-auth"]; ok {
+		jwt := input.Plugins["jwt-auth"].(map[string]interface{})
+		jwt["exp"] = 86400
```

Comment:
why add exp = 86400？

Reply:
Because there is a problem with the default value setting of the current jsonschema, a temporary solution

Reply:
Please add issue for it